lib/hypervisor/hv_kvm/__init__.py

```diff
@@ -1529,6 +1521,23 @@ def _GenerateKVMRuntime(self, instance, block_devices, startup_paused,
       else:
         kvm_cmd.extend(["-nographic"])
 
+    # As requested by music lovers
+    if hvp[constants.HV_SOUNDHW]:
+      soundhw = hvp[constants.HV_SOUNDHW]
+      if self._SOUND_RE.search(kvmhelp):
+        kvm_cmd.extend(["-soundhw", soundhw])
+      else:
+        # Qemu versions >= 7.1 do not support -soundhw anymore
+        # also, we need to pick a host backend/driver. we'll use
+        # spice if that is configured, otherwise there will be
+        # no sound output
+        if spice_bind:
+          driver = "spice"
+        else:
+          driver = "none"
+        kvm_cmd.extend(["-audio", "driver=%s,model=%s,id=soundhw"
+                        % (driver, soundhw)])
+
```

Comment:
If you are already editing something here, please use format strings right away:
```suggestion
        kvm_cmd.extend(["-audio", "driver={},model={},id=soundhw"
                               .format(driver, soundhw)])
```

Reply:
done